packages/core/lib/commands/migrate/setUpDryRunEnvironmentThenRunMigrations.js

```diff
@@ -24,7 +23,15 @@ module.exports = async function (config) {
     prefix: "migrate-dry-run-"
   }).name;
 
-  await promisify(copy)(config.contracts_build_directory, temporaryDirectory);
+  // copy returns an error if there is an error, undefined if successful
+  const error = await copy(
+    config.contracts_build_directory,
+    temporaryDirectory,
+    { overwrite: true }
+  );
+  if (error) {
```

Comment:
Rather than having this check here, might it make more sense to put this check inside the `copy` function itself?  You didn't put it on the other invocations, but I'm assuming they should probably have it, right?  (And if they shouldn't, they should probably use a `try`/`catch` to explicitly swallow the error rather than implicitly swallowing it like they do in this version.)

Reply:
Ah yes, this is a good idea.